OpenEdXMobile/src/main/java/org/edx/mobile/view/MyCoursesListFragment.java

```diff
@@ -257,7 +258,8 @@ public void onClick(View v) {
         footer.courseNotListedTv.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                environment.getRouter().showWebViewActivity((getActivity()), getString(R.string.course_not_listed_file_name), null);
+                AlertDialogFragment.newInstance(null, getString(R.string.cant_find_course_prompt), null)
```

Comment:
Imo we should rely on that function while we are in an activity's context. For future proofing, this fragment might reside in an activity that might not be a subclass of `BaseFragmentActivity`, so its better to use `AlertDialogFragment` directly here.